lisa/tools/kdump.py

```diff
@@ -310,6 +310,14 @@ def enable_kdump_service(self) -> None:
         service = self.node.tools[Service]
         service.enable_service("kdump")
 
+    def restart_kdump_service(self) -> None:
+        """
+        This method restart the kdump service. If distro has a different kdump service
+        name, need override it.
+        """
+        service = self.node.tools[Service]
+        service.restart_service("kdump")
```

Comment:
how about enable it by default? will it be started automatically after reboot?

Reply:
Most of the time, it will be started automatically after a reboot. 
I see one failure is caused by the kdump service being inactive after a reboot. The inactive may be caused by the alternative crash path /mnt/crash that hasn't been mounted when the kdump service started. After restarting the service, the kdump service is active. 

After the double check, restarting kdmp service should not be added here, which will hide the kdump inactive issue after a reboot. I have deleted this commit.